src/components/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ChangeEvent, InputHTMLAttributes, useId, useState } from 'react';
+import styled from '@emotion/styled';
+
+import IconCheckbox from '../icon/IconCheckbox';
+
+import useDidUpdate from '@/hooks/life-cycle/useDidUpdate';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+  onCheck: () => void;
+  onUncheck: () => void;
```

Comment:
```suggestion
interface Props extends InputHTMLAttributes<HTMLInputElement> {
  onCheck?: () => void;
  onUncheck?: () => void;
```

이것도 partial로 선언하면 어떨까요 ?!